ci/pipeline.yml.erb

```diff
@@ -24,11 +24,11 @@ resource_types:
     tag: latest
 
 resources:
-- name: ubuntu-16.04
+- name: ubuntu-18.04
```

Comment:
How about making "ubuntu-18.04" be variable?

Reply:
Do you mean that `ubuntu-18.04` should be changed to less specific name such as `base-image`?

Reply:
No,  we are using ERB, so we can use ruby variable.

Reply:
Ah, OK. I understand.  I'll do it.

Reply:
done. 5f9e1b1